### DISCUSSION TITLE
Google Season of Docs 2023 CNCF Project Proposals

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#843] [Announcements]
Hello!

CNCF has participated in the Google Season of Docs several times over the [last few years](https://github.com/cncf/mentoring/tree/main/seasonofdocs), and now we are ready to start the process  of applying to the program as an organization this year.
 
All CNCF projects are eligible to make proposals for the Google Season of Docs. Please start working on the project proposals, and submit them via pull request: https://github.com/cncf/mentoring/tree/main/seasonofdocs#2023-project-proposals

Please make a note of the [proposal template](https://developers.google.com/season-of-docs/docs/org-proposal-template). We  have yet to determine if we will be accepted as an org and, if we do, how many slots we will be given, so the quality of proposal is important.

The deadline to submit CNCF proposals is March 14, 2023 (though we would be happy for project ideas to come in sooner than that if possible).

[Google Season of Docs 2023 Timeline](https://developers.google.com/season-of-docs/docs/timeline)

Thanks!

**Comment by MahadMuhammad** [comment]


**Reply by aliok** [reply]
@drmahad This announcement from Nate is for the project maintainers. They will create proposals for Google and then these proposals will be evaluated. Then, projects will "hire" professional technical writers. This program is not about teaching technical writing. It is more about getting technical writers involved into open source projects.

From GSoD website:

> Google Season of Docs provides support for open source projects to improve their documentation and gives professional technical writers an opportunity to gain experience in open source. Together we raise awareness of open source, of docs, and of technical writing.

**Reply by aliok** [reply]
@drmahad It is a bit hard for me to give you direction on getting started into technical writing. It is a profession.

Again, GSoD program is not about teaching people technical writing. It is more about getting professional technical writers involved into open source.

**Comment by xmulligan** [comment]
Should we submit through CNCF or apply directly ourselves as a project? Can CNCF have more than one project?

**Reply by nate-double-u** [reply]
Hi @xmulligan, sorry about the delayed response here. Short answer is I'm not sure yet. We've discussed this in our [Mentoring WG meeting today](https://hackmd.io/9tmmHWSmRaiPkmnur-Q6OA?view#February-28-2023), and are looking to find out.

**Reply by nate-double-u** [reply]
We've actually put a post on the discussion boards as we're a little confused as well: https://groups.google.com/g/season-of-docs-discuss/c/PxTWKQfblLc

**Reply by nate-double-u** [reply]
OK, here's the response from a Google Season of Docs program manager:

> We still ask that each project apply separately; if a project is selected, they can choose CNCF as their payment option with Open Collective when we begin setting up grant payments.
> 
> Projects can feel free to add in their application that payments would be handled via CNCF but that's not a factor in our decision making for what projects to accept.

This means that each project should be making their own application. We're happy to help review these, but each project needs to make their own submission.

**Comment by nate-double-u** [comment]
Exciting news: [Flux](https://fluxcd.io/contributing/docs/google-season-of-docs-2023/) and [WasmEdge](https://github.com/WasmEdge/GSoD2023) have been accepted into Google Season of Docs 2023, making the cut among the 13 selected projects! We're eager to see the collaboration between the projects and technical writers, enhancing documentation and user experience. Stay tuned for updates, and congrats to Flux and WasmEdge! 🎉

The full list of 13 participating open source organizations for Season of Docs 2023 is now available at https://developers.google.com/season-of-docs/docs/participants/.

**Reply by aliok** [reply]
Congrats @dholbach and @alabulei1 🎉

**Reply by alabulei1** [reply]
Thanks for your instruction! @aliok and @nate-double-u

**Reply by dholbach** [reply]
Thanks for your help @nate-double-u and @aliok - much much appreciated!

**Reply by alabulei1** [reply]
Hi @nate-double-u & @aliok,

I have a small question for GSoD. Do we need to set up an Open Collective account by ourselves, or will CNCF cover this? I'm fine with either option; I just want to ensure that everything is taken care of and doesn't add extra burdens for you. I can tell that you're very busy with the new version of LF Mentorship. Thank you for this awesome work!

**Reply by aliok** [reply]
@nate-double-u can answer the money related questions as he's a CNCF employee but he's off this week.

@alabulei1 What is the deadline to set it up?

**Reply by dholbach** [reply]
May 3, 18UTC.

We are in the same boat.

Cc @hiddeco @kingdonb

**Reply by aliok** [reply]
There's still 3 days next week after Nate returns. With good coordination, it might work I guess?

Not sure if setting up an Open Collective account is easy or hard. You might want to setup on your own as a backup anyway.

**Reply by aliok** [reply]
I've pinged Nate now.

**Reply by nate-double-u** [reply]
I'm reaching out to some folks internally, I'll let you all know when I hear back.

(Bookmarking the payments page for reference https://developers.google.com/season-of-docs/docs/org-payments)

**Reply by dholbach** [reply]
Do you have any recommendations for us?

**Reply by nate-double-u** [reply]
@dholbach I'm just back from vacation this morning -- I'm still working with the folks internally on this. I hope to have a better answer soon.

**Reply by nate-double-u** [reply]
Update, the CNCF will be setting up an Open Collective account -- I'll report back when we have.

**Reply by dholbach** [reply]
Thanks @nate-double-u

**Reply by nate-double-u** [reply]
Hi @alabulei1 & @dholbach I have sent you both an email with the form information you need. Please reach out to me via CNCF Slack or by email if you haven't received it (@alabulei1 I used the email listed on your github profile page).
